cli/src/semgrep/constants.py

```diff
@@ -40,6 +42,12 @@ class EngineType(Enum):
     def is_pro(self) -> bool:
         return self.value >= EngineType.PRO.value
 
+    def to_engine_kind(self) -> out.EngineKind:
```

Comment:
I wonder how your work interacts with stuff like:
```
        if any(match.severity == RuleSeverity.EXPERIMENT for match in new_ignored):
            logger.info("Some experimental rules were run during execution.")
```
(scans.py: 318)

Reply:
this is just the engine kind convertor, surely that shouldn't affect the severity, right?

Reply:
Oh, I more meant on an architectural level, how this affects the readability